bcdaworker/main.go

```diff
@@ -218,20 +218,25 @@ func writeBBDataToFile(ctx context.Context, bb client.APIClient, db *gorm.DB, cm
 	failed := false
 
 	for _, beneID := range jobArgs.BeneficiaryIDs {
-		blueButtonID, err := beneBBID(beneID, bb, db)
-
-		if err != nil {
-			handleBBError(ctx, err, &errorCount, fileUUID, fmt.Sprintf("Error retrieving BlueButton ID for cclfBeneficiary %s", beneID), jobArgs.ID)
-		} else {
-			b, err := bundleFunc(blueButtonID)
+		bundle := func() (string, error) {
```

Comment:
Do you need this to be of type `func()`? If you just need the result, you can call the anonymous function directly and get the result.

```
errMsg, err := func() (string, err) {
...
}()
```
The last () will invoke the  function